src/components/SecondaryPanes/Frames/FrameMenu.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import { showMenu } from "devtools-launchpad";
+import { copyToTheClipboard } from "../../../utils/clipboard";
+import type { LocalFrame } from "./types";
+
+function formatMenuElement(label, accesskey, click, disabled = false) {
+  return {
+    id: "node-menu-copy-source",
```

Comment:
we need to have unique IDs.

I'm not sure how much this API buys us actually. Perhaps, we establish a convention where we give it a label, and it can make an ID and accessKey?

// devtools-modules/src/menu.js
```js
export function createItem({label, disabled = false, click }) {
  const id = `node-menu-${kebabCase(label)}`

  return {
    label,
    id,
   ...
  }
}
```

https://lodash.com/docs/4.17.4#kebabCase